lib/policy_machine.rb

```diff
@@ -69,7 +69,7 @@ def add_association(user_attribute_pe, operation_set, object_attribute_pe)
   # TODO: Parallelize the two component checks
   def is_privilege?(user_or_attribute, operation, object_or_attribute, options = {})
     is_privilege_ignoring_prohibitions?(user_or_attribute, operation, object_or_attribute, options) &&
-      !is_privilege_ignoring_prohibitions?(user_or_attribute, PM::Prohibition.on(operation), object_or_attribute, options)
+      (options[:ignore_prohibitions] || !is_privilege_ignoring_prohibitions?(user_or_attribute, PM::Prohibition.on(operation), object_or_attribute, options))
```

Comment:
Oh, that might actually be a legacy of an earlier implementation. Let me see if I can remove it.